### PR TITLE
T/ckeditor5/1151: Integrated the text alignment feature with different editor content directions

### PR DESCRIPTION
**Pull request by oleq**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Feature: Integrated the text alignment feature with different editor content directions (LTR and RTL). See ckeditor/ckeditor5#1151.

---

### Additional information

A piece of https://github.com/ckeditor/ckeditor5/pull/1881.

### CODE REVIEW

File: docs/features/text-alignment.md [hunk 1]
```diff
@@ -13,7 +13,11 @@ The {@link module:alignment/alignment~Alignment} feature enables support for tex
 
 ## Configuring alignment options
 
-It is possible to configure which alignment options are available in the editor by setting the {@link module:alignment/alignment~AlignmentConfig#options `alignment.options`} configuration option. You can choose from `'left'`, `'right'`, `'center'` and `'justify'`;  note that `'left'` should always be included.
+It is possible to configure which alignment options are available in the editor by setting the {@link module:alignment/alignment~AlignmentConfig#options `alignment.options`} configuration option. You can choose from `'left'`, `'right'`, `'center'` and `'justify'`.
+
+<info-box>
+	Note that `'left'` or `'right'` should always be included for <abbr title="left–to–right">LTR</abbr> and <abbr title="right-to-left">RTL</abbr> content, respectively. Learn more about {@link features/ui-language#setting-the-language-of-the-content configuring language of the editor content}.
```
Comment (Reinmar) [inline docs/features/text-alignment.md]: I had to thing trice to understand this. Could you write it in a more direct way? Like:

> Note that the `'left'` option must be included if you set up the editor content to be LTR. Respectively, the `'right'` option should be included if you bla bla bla.

**Comment by Reinmar** [comment]
I think that the defult order of these items should be different in RTL ("align right" should be first):

image

**Review by Reinmar — CHANGES_REQUESTED** [review]
As commented.

**Comment by oleq** [comment]
> I think that the defult order of these items should be different in RTL ("align right" should be first):

It's controlled via `AlignmentConfig#options` and totally up to the developers.

File: docs/features/text-alignment.md (hunk 1 above)
Comment (oleq) [reply docs/features/text-alignment.md]:
```suggestion
	Note that the `'left'` option should always be included for the <abbr title="left–to–right">LTR</abbr> content. Similarly, the `'right'` option should always be included for the <abbr title="right-to-left">RTL</abbr> content. Learn more about {@link features/ui-language#setting-the-language-of-the-content configuring language of the editor content}.
```

**Comment by Reinmar** [comment]
> It's controlled via `AlignmentConfig#options` and totally up to the developers.

Doh, I thought there's a default value of this config. Don't you think there should be?

**Review by Reinmar — APPROVED** [review]

**Comment by oleq** [comment]
> Doh, I thought there's a default value of this config. Don't you think there should be?

The feature buttons are configured via editor config https://ckeditor.com/docs/ckeditor5/latest/features/text-alignment.html#configuring-alignment-options. I'm not sure how this would work if we enforced some button order on our side.

**Pull request merged**